packages/rrweb/src/replay/index.ts

```diff
@@ -1521,9 +1508,6 @@ export class Replayer {
       if (mutation.nextId) {
         next = mirror.getNode(mutation.nextId);
       }
-      if (nextNotInDOM(mutation)) {
-        return queue.push(mutation);
-      }
```

Comment:
`nextId` when recording is found using `nextSibling` 
so I can't think why we'd wait for it here...

adding node X to parent Y doesn't need to queue until it's sibling is present first, surely